<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Safe</class>
 <widget class="QWidget" name="Safe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Safe</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(246, 246, 247);</string>
  </property>
  <widget class="QStackedWidget" name="Stacked">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>32</y>
     <width>800</width>
     <height>447</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="MainPage">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QFrame" name="MainImage">
     <property name="geometry">
      <rect>
       <x>303</x>
       <y>46</y>
       <width>195</width>
       <height>162</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/res/1.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="Title">
     <property name="geometry">
      <rect>
       <x>335</x>
       <y>230</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 81 13pt &quot;Raleway&quot;;</string>
     </property>
     <property name="text">
      <string>Safe Guider</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>260</y>
       <width>321</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;나눔스퀘어&quot;;
color: #C3CCCD</string>
     </property>
     <property name="text">
      <string> Safe Guider는 화재 발생 원인이 존재하는지 알려줍니다. 
 또한, 화재 발생시 대피 경로를 알려주며, 탈출이 어려울 시
     에는 위치를 추적하여 내 위치를 알려줄 수 있습니다.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BluePair">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>360</y>
       <width>260</width>
       <height>48</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(109, 116, 204);
border-radius: 23px;
font: 75 10pt &quot;나눔스퀘어 Bold&quot;;
color: rgb(255,255,255);</string>
     </property>
     <property name="text">
      <string>블루투스 페어링 연결</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SetLoc">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>360</y>
       <width>260</width>
       <height>48</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 23px;
font: 75 10pt &quot;나눔스퀘어 Bold&quot;;
color: rgb(0,0,0);
</string>
     </property>
     <property name="text">
      <string>직접 위치 등록하기</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="SetAdrPage">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QFrame" name="ConnectImage">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>283</width>
       <height>186</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/res/undraw_confirmation_2_uy_0.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="BlueRect">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>78</y>
       <width>7</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(109, 116, 204);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>139</x>
       <y>71</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;나눔스퀘어 Bold&quot;;</string>
     </property>
     <property name="text">
      <string>직접 위치 설정하기</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>446</x>
       <y>167</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 9pt &quot;나눔스퀘어 Bold&quot;;</string>
     </property>
     <property name="text">
      <string>집주소를 입력해주세요</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>446</x>
       <y>197</y>
       <width>300</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어</family>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(248, 248, 248);
font: 8pt &quot;나눔스퀘어&quot;;
border-radius: 5px</string>
     </property>
     <property name="placeholderText">
      <string> 세부 주소를 써주세요</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Cancelsetloca">
     <property name="geometry">
      <rect>
       <x>597</x>
       <y>257</y>
       <width>121</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20px;
font: 75 9pt &quot;나눔스퀘어 Bold&quot;;
color: rgb(0,0,0);
</string>
     </property>
     <property name="text">
      <string>취소하기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="setAddrbtn">
     <property name="geometry">
      <rect>
       <x>466</x>
       <y>257</y>
       <width>121</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(109, 116, 204);
border-radius: 20px;
font: 75 9pt &quot;나눔스퀘어 Bold&quot;;
color: rgb(255,255,255);</string>
     </property>
     <property name="text">
      <string>등록하기</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="BluePage">
    <widget class="QFrame" name="ConnectImage_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>283</width>
       <height>186</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/res/undraw_confirmation_2_uy_0.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="SelectedDivice">
     <property name="geometry">
      <rect>
       <x>446</x>
       <y>177</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어</family>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(248, 248, 248);
font: 8pt &quot;나눔스퀘어&quot;;
border-radius: 5px</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>139</x>
       <y>71</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;나눔스퀘어 Bold&quot;;</string>
     </property>
     <property name="text">
      <string>블루투스 페어링 연결</string>
     </property>
    </widget>
    <widget class="QLabel" name="BlueRect_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>78</y>
       <width>7</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(109, 116, 204);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>446</x>
       <y>147</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 9pt &quot;나눔스퀘어 Bold&quot;;</string>
     </property>
     <property name="text">
      <string>등록기기</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>446</x>
       <y>216</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 9pt &quot;나눔스퀘어 Bold&quot;;</string>
     </property>
     <property name="text">
      <string>기타기기</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>70</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 9pt &quot;나눔스퀘어 Bold&quot;;</string>
     </property>
     <property name="text">
      <string>Bluetooth</string>
     </property>
    </widget>
    <widget class="QListWidget" name="DeviceList">
     <property name="geometry">
      <rect>
       <x>446</x>
       <y>240</y>
       <width>261</width>
       <height>96</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(248, 248, 248);
font: 8pt &quot;나눔스퀘어&quot;;
border-radius: 5px;
margin: 2px;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="TempPage">
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>780</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 8px;
font: 75 9pt &quot;나눔스퀘어 Bold&quot;;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="Temp">
     <property name="geometry">
      <rect>
       <x>219</x>
       <y>105</y>
       <width>167</width>
       <height>246</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/res/24.png);
font: 75 25pt &quot;나눔스퀘어 Bold&quot;;
color: rgb(255,255,255);</string>
     </property>
     <property name="text">
      <string>33°
</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>123</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 81 9pt &quot;Raleway&quot;;
color: rgb(255,255,255);
background-color: rgba(255, 255, 255, 0);
border-radius: 5px</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>Temperature</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>140</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 7pt &quot;나눔스퀘어&quot;;
color: ;
color: rgb(197, 197, 197);
background-color: rgba(255, 255, 255, 0);
border-radius: 5px</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>온도</string>
     </property>
    </widget>
    <widget class="QLabel" name="Temp_2">
     <property name="geometry">
      <rect>
       <x>411</x>
       <y>106</y>
       <width>167</width>
       <height>246</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/res/25.png);
font: 75 25pt &quot;나눔스퀘어 Bold&quot;;
color: rgb(255,255,255);</string>
     </property>
     <property name="text">
      <string>22%
</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>462</x>
       <y>123</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 81 9pt &quot;Raleway&quot;;
color: rgb(255,255,255);
background-color: rgba(255, 255, 255, 0);
border-radius: 5px</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>Humidity </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>483</x>
       <y>140</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 7pt &quot;나눔스퀘어&quot;;
color: ;
color: rgb(197, 197, 197);
background-color: rgba(255, 255, 255, 0);
border-radius: 5px</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>습도</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SaveMe">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>380</y>
       <width>200</width>
       <height>47</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255,0,0);
border-radius: 23px;
font: 75 10pt &quot;나눔스퀘어 Bold&quot;;
color: rgb(255,255,255);</string>
     </property>
     <property name="text">
      <string>구조 요청</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>252</x>
       <y>307</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 5pt &quot;Raleway&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>ⓒ Team ESC - Safe Guider</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>444</x>
       <y>308</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 5pt &quot;Raleway&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>ⓒ Team ESC - Safe Guider</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="HelpPage">
    <widget class="QPushButton" name="SaveMe_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>380</y>
       <width>200</width>
       <height>47</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255,0,0);
border-radius: 23px;
font: 75 10pt &quot;나눔스퀘어 Bold&quot;;
color: rgb(255,255,255);</string>
     </property>
     <property name="text">
      <string>구조 요청</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>780</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 8px;
font: 75 9pt &quot;나눔스퀘어 Bold&quot;;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>448</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 100);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>75</y>
       <width>360</width>
       <height>250</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/res/28.png);
border-radius: 15px;
font: 75 14pt &quot;나눔스퀘어 Bold&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="Call">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>270</y>
       <width>360</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);
font: 75 11pt &quot;나눔스퀘어 Bold&quot;;
border-radius: 1px;
color: rgb(255,255,255);
</string>
     </property>
     <property name="text">
      <string>요청하기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Back">
     <property name="geometry">
      <rect>
       <x>553</x>
       <y>88</y>
       <width>11</width>
       <height>11</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/res/delete.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="FirePage">
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>780</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 8px;
font: 75 9pt &quot;나눔스퀘어 Bold&quot;;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SaveMe_3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>380</y>
       <width>200</width>
       <height>47</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255,0,0);
border-radius: 23px;
font: 75 10pt &quot;나눔스퀘어 Bold&quot;;
color: rgb(255,255,255);</string>
     </property>
     <property name="text">
      <string>구조 요청</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Back_2">
     <property name="geometry">
      <rect>
       <x>553</x>
       <y>88</y>
       <width>11</width>
       <height>11</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/MainImage/C:/Users/user/Desktop/Image/delete.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>75</y>
       <width>360</width>
       <height>250</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/res/29.png);
border-radius: 15px;
font: 75 14pt &quot;나눔스퀘어 Bold&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>448</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 100);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Call_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>270</y>
       <width>360</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);
font: 75 11pt &quot;나눔스퀘어 Bold&quot;;
border-radius: 1px;
color: rgb(255,255,255);
</string>
     </property>
     <property name="text">
      <string>구조 요청</string>
     </property>
    </widget>
    <zorder>SaveMe_3</zorder>
    <zorder>label_18</zorder>
    <zorder>label_17</zorder>
    <zorder>Back_2</zorder>
    <zorder>label_16</zorder>
    <zorder>Call_2</zorder>
   </widget>
  </widget>
  <widget class="QLabel" name="Bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(109, 116, 204);
font: 25 9pt &quot;Raleway&quot;;
color: #ffffff</string>
   </property>
   <property name="text">
    <string>   Safe Guider</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>7</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);;
font: 75 11pt &quot;나눔스퀘어 Bold&quot;;
color: white;
</string>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
